HW_3/task_3.py

```diff
@@ -0,0 +1,24 @@
+#"""
+#Задайте список из вещественных чисел, округленных до сотых.
+#Найдите разницу между максимальным и минимальным значением дробной части элементов.
+
+#Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+#Вывод: значение типа <float>
+
+#Пример:
+#[1.1, 1.2, 3.1, 5, 10.01]
+#2.0
+#"""
+
+def minmax (list):
+    max_min = []
+    for i in range(len(list)):
+        if list[i]%1 != 0:
+            max_min.append(list[i]%1)
```

Comment:
Функция `range` использовать необязательно проходитесь прямо по элементам списка А еще, чтобы не делать одно и то же действие дважды лучше поместить результат этого действия в переменную
```suggestion
    for i in range(len(list)):
        res = i % 1
        if res != 0:
            max_min.append(res)
```